cmd/repo.go

```diff
@@ -155,13 +157,28 @@ func newRepoGenkeyCmd(env *meta.Environment) *cobra.Command {
 		Short: "Generate a new key pair",
 		Long:  `Generate a new key pair that can be used to sign components.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			privPath := env.Profile().Path("private.pem")
+			pubPath := env.Profile().Path("public.pem")
```

Comment:
I don't think we should store the public to the disk, because we can obtain the public key from private key directly.

Reply:
Yes